### ISSUE TITLE
emercoin.i2p

### ISSUE THREAD

**Issue by blade10101** [emercoin/docs#14]
the site is dead

**Comment by maxihatop** [comment]
Yes, you right. Site owner created service, but because of i2p community ignored it, he just discontinued to support it.
However, site is **not needed** to download ip2-records from the blockchain, or register a new  i2 record.
All records in the blockchain are still exist, and you can see or use them, for example:

-  https://explorer.emercoin.com/nvs/i2p/emercoin.i2p/
-  https://explorer.emercoin.com/nvs/i2p/emc.i2p/

Also, you can generate actual "hosts.txt" for i2p from the running blockchain node anytime.
CLI command to download i2p-records from blockahin into JSON-file:

$ emercoin-cli name_filter ^i2p: >/tmp/i2p.json

Thereafter, you can convert that JSON to a local hosts.txt, and use it.

Also, this document describes, how to register i2p domain within Emer blockchain:

https://github.com/emercoin/docs/blob/master/en/020_Blockchain_Services/030_EmerDNS/031_I2P_Domain_Name_Registration_based_on_blockchain.md
